pyscriptjs/src/pyodide.ts

```diff
@@ -30,17 +29,30 @@ export class PyodideRuntime extends Runtime {
         this.lang = lang;
     }
 
+    /**
+     * Although `loadPyodide` is used below,
+     * notice that it is not imported i.e.
+     * import { loadPyodide } from 'pyodide';
+     * is not used at the top of this file.
+     *
+     * This is because, if it's used, loadPyodide
+     * behaves mischievously i.e. it tries to load
+     * `pyodide.asm.js` and `pyodide_py.tar` but
+     * with paths that are wrong such as:
+     *
+     * http://127.0.0.1:8080/build/pyodide_py.tar
+     * which results in a 404 since `build` doesn't
+     * contain these files and is clearly the wrong
+     * path.
+     */
```

Comment:
By reading this comment it's not clear whether this weird behavior is caused by a pyodide bug or by pyscript using it wrongly. Do we know that?

Reply:
In general, I am a bit skeptical to introduce workarounds without fully understanding what's going on, but at the same time you probably already spent enough time debugging this that it's a good idea to move on.

Reply:
Yeah, I re-introduced my changes from the Runtime Refactor PR to figure out at what point it broke. Turns out it was this line and we weren't using this line before the Runtime Refactor PR. So in some sense, I restored to the original state.